.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 target/
+SensitiveData.ts
```

Comment:
seems like theres some merging/rebasing that needs to be done.

one book I really enjoyed on git was called [git from the bottom up](https://jwiegley.github.io/git-from-the-bottom-up/), it explains git at a deep level (but is less a practical guide).